coverage_comment/coverage.py

```diff
@@ -139,7 +140,10 @@ def extract_info(data) -> Coverage:
                 info=CoverageInfo(
                     covered_lines=file_data["summary"]["covered_lines"],
                     num_statements=file_data["summary"]["num_statements"],
-                    percent_covered=file_data["summary"]["percent_covered"] / 100,
+                    percent_covered=decimal.Decimal(
+                        str(file_data["summary"]["percent_covered"])
```

Comment:
If we're putting precision in this thing, shouldn't we recompute percent_covered as `covered_lines/num_statements` ? I'm guessing we may inherit rounding errors otherwise.

Reply:
Yes, that is a very good point :+1:.
Are you sure that this is the calculation formula that is also used internally? Don't we still have to include the branches somehow? Because otherwise one of our [test cases](https://github.com/py-cov-action/python-coverage-comment-action/blob/v3/tests/conftest.py#L88) seems to be wrong? 
I have already tried to find documentation for this, but I was not really successful

Reply:
`num_statements+num_statements` ?

Reply:
😄 Copied the wrong bit of code: `(covered_lines+covered_branches)/(num_statements+num_branches)`

Reply:
:+1: